draft-irtf-cfrg-vdaf.md

```diff
@@ -1190,6 +1190,252 @@ how the nonces are chosen, but security requires that the nonces be unique. See
 execution of a VDAF requires the application to instantiate secure channels
 between each of the protocol participants.
 
+## Communication Patterns for Preparation {#vdaf-prep-comm}
```

Comment:
All of this is written in terms of VDAFs. Does that mean this is undefined for DAFs?

Reply:
Yes.